app/code/Magento/Inventory/Test/Integration/Indexer/SourceIndexerTest.php

```diff
@@ -13,6 +13,9 @@
 use Magento\TestFramework\Helper\Bootstrap;
 use PHPUnit\Framework\TestCase;
 
+/**
+ * @magentoAppArea adminhtml
```

Comment:
Indexer test must work in any areas
If need you could use trick with `is secure area`